domain/src/main/java/org/oppia/android/domain/state/RetriveUserAnswer.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.android.domain.state
+
+import org.oppia.android.app.model.UserAnswer
+
+object RetriveUserAnswer {
```

Comment:
This should be a Dagger singleton (see other ``@Singleton`` marked classes in the codebase).